src/PluginManager.cpp

```diff
@@ -2416,14 +1434,6 @@ void PluginManager::CheckForUpdates(bool bFast)
    return;
 }
 
-bool PluginManager::ShowManager(wxWindow *parent, EffectType type)
-{
-   CheckForUpdates();
-
-   PluginRegistrationDialog dlg(parent, type);
-   return dlg.ShowModal() == wxID_OK;
-}
-
```

Comment:
This internal, PluginManager class no longer knows about the user interface that gives access to it.  So this function is elsewhere now.